kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -127,6 +127,15 @@
   private static final String AVRO_DATA_CACHE_SIZE_DOC =
       "The size of the cache to use when converting schemas from Avro to Kafka Connect";
 
+  public static final String CONVERT_DOUBLE_SPECIAL_VALUES_CONFIG =         "convertDoubleSpecialValues";
+  public static final ConfigDef.Type CONVERT_DOUBLE_SPECIAL_VALUES_TYPE =   ConfigDef.Type.BOOLEAN;
+  public static final Boolean CONVERT_DOUBLE_SPECIAL_VALUES_DEFAULT =       true;
```

Comment:
I think this should be false by default. It is kind of surprising behavior, so I'd rather have the loads fail, and have the operator investigate and discover this config, rather than have things auto-convert without the operator knowing.

Reply:
Plus one for this; it also preserves backwards compatibility so existing users of the connector won't experience unexpected changes in the data it's writing.